internal/app/machined/main.go

```diff
@@ -106,6 +106,8 @@ func main() {
 
 	// Start the boot sequence in a go routine so that we can list for events.
 	go func() {
+		defer recovery()
+		log.Println(err)
```

Comment:
not sure what this is supposed to print? outer value of `err`? looks strange

Reply:
ahh I think your right, I can fix